lib/src/components/NxSmallVulnerabilityCounter/NxSmallVulnerabilityCounter.scss

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@use "sass:math";
+@use "../../scss-shared/nx-text-helpers";
+
+@mixin counter-colors {
+  @each $severity in critical, high, medium, low, none {
+    &--#{$severity} {
+      background-color: var(--nx-color-vulnerability-#{$severity});
+      color: var(--nx-color-vulnerability-#{$severity}-text);
+      border: 1px solid var(--nx-color-vulnerability-#{$severity}-border);
+    }
+  }
+}
+
+.nx-small-vulnerability-counter {
+  $height: 25px;
+  position: relative;
+  border-radius: 2px;
+  font-size: var(--nx-font-size-s);
+  font-weight: 600;
+  height: $height;
+  line-height: $height;
+  padding-left: var(--nx-spacing-base);
+  padding-right: var(--nx-spacing-base);
+  text-align: center;
+
+  &__severity {
+    //invisible but still read by the screenreader
+    @include nx-text-helpers.visually-hidden;
+  }
+
+  @include counter-colors;
+
+  // Used to set the width of the counter based on its max text content
+  &__sizer {
+    display: block;
+    height: 0;
+    overflow: hidden;
+    visibility: hidden;
+  }
+
+  &--zero {
+    background-color: var(--nx-color-small-vulnerability-counter-zero-background);
+    color: var(--nx-color-small-vulnerability-counter-zero-text);
+    border: 1px solid var(--nx-color-small-vulnerability-counter-zero-border);
+  }
+}
+
+.nx-small-vulnerability-counter-container {
+  display: inline-flex;
+  gap: var(--nx-spacing-2x);
+  margin-left: var(--nx-spacing-2x);
+  margin-right: var(--nx-spacing-2x);
+
+  &--no-max {
+    display: inline-grid;
+    grid-auto-columns: 1fr;
+    grid-auto-flow: column;
+
+    .nx-small-vulnerability-counter {
+      box-sizing: border-box;
+      text-align: center;
+      width: 100%;
+
+      // using the counter-colors mixin will override the zero styling in this use case
+      @include counter-colors;
```

Comment:
The spec doesn't call this out.  Check with @JasonDSwearingen that this is desired for this component just like `NxSmallThreatCounter`.

Reply:
> The spec doesn't call this out. Check with @JasonDSwearingen that this is desired for this component just like `NxSmallThreatCounter`.

I checked with the designers, it should behave the same with threat counter so this is correct.
They will work on specifying this on the figma side

Reply:
I was asked about this, and this is correct. It just needs documented correctly in Kaleido.

Reply:
> I was asked about this, and this is correct. It just needs documented correctly in Kaleido.

I still don't see this documented in Kaleido, either in the new component nor in `NxSmallThreatCounter`.  To be clear, the behavior implemented by this line is that when you're in the "no max" mode depicted at the bottom of the mockup, counters with a zero will maintain their normal color rather than switching to the "zero" color.